stdlib/REPL/src/LineEdit.jl

```diff
@@ -1477,22 +1477,30 @@ end
 
 current_word_with_dots(s::MIState) = current_word_with_dots(buffer(s))
 
+previous_active_module = Main
```

Comment:
Maybe could be a typed global? (`::Module`)?

Reply:
https://github.com/JuliaLang/julia/pull/52800